static/docs/commands-reference/repro.md

```diff
@@ -15,25 +15,83 @@ name is omitted, `Dvcfile` will be used by default.
     positional arguments:
         target                DVC file to reproduce.
 
-    optional arguments:
```

Comment:
minor: `## Synopsis` is missing

Reply:
before ```usage

Reply:
Let's also move `DVC file (`target`) can have any name followed by the `.dvc` suffix. If file
name is omitted, `Dvcfile` will be used by default.` into description?

Reply:
Thank you

Reply:
Have updated the file to match